e2e/pages/ShowPage.js

```diff
@@ -0,0 +1,29 @@
+import { By, until } from 'selenium-webdriver';
+
+module.exports = (url) => (driver) => ({
+    elements: {
+        appLoader: By.css('.app-loader'),
+        field: (name) => By.css(`.aor-field-${name} > div > span`),
```

Comment:
when inline and with a single argument, remove the parentheses around the argument of an anonymous function.

```
field: name => By.css(`.aor-field-${name} > div > span`),
```